### PR TITLE
updated ReadMe

### PR DESCRIPTION
**Pull request by Benjameads**
Created ReadMe.md and updated markdown file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# MtnMentalHealth
```
Comment (Septimis) [inline README.md:1]: Why did you delete this and put it back?

Comment (Benjameads) [reply README.md:1]: Because I was learning the system.
